pkgs/development/libraries/tpm2-tss/default.nix

```
@@ -63,6 +63,11 @@ stdenv.mkDerivation rec {
       --replace '@PREFIX@' $out/lib/
     substituteInPlace ./bootstrap \
       --replace 'git describe --tags --always --dirty' 'echo "${version}"'
+    for src in src/tss2-tcti/tcti-libtpms.c test/unit/tcti-libtpms.c; do
+      substituteInPlace "$src" \
+        --replace '"libtpms.so"' '"${libtpms.out}/lib/libtpms.so"' \
+        --replace '"libtpms.so.0"' '"${libtpms.out}/lib/libtpms.so.0"'
```

```suggestion
        --replace-fail '"libtpms.so"' '"${libtpms.out}/lib/libtpms.so"' \
        --replace-fail '"libtpms.so.0"' '"${libtpms.out}/lib/libtpms.so.0"'
```
we should also replace the others to make sure that things match

---

We really should set this to notice when the shared objects no longer match.